lib/core/lib/ops/userOps.js

```diff
@@ -513,6 +551,43 @@ function buildUserOps (db/*, config */) {
       })
     },
 
+    /**
+     * List the teams to which the user belongs to
+     * Does not go recursively to parent teams
+     *
+     * @param  {Object}   params { id, organizationId, limit, page }
+     * @param  {Function} cb
+     */
+    listUserTeams: function listUserTeams ({ id, organizationId, page = 1, limit }, cb) {
+      Joi.validate({ id, organizationId, page, limit }, validationRules.listUserTeams, function (err) {
+        if (err) return cb(Boom.badRequest(err))
+
+        const pageLimit = limit || config.get('authorization.defaultPageSize')
+        const offset = (page - 1) * pageLimit
+
+        const job = {
+          id,
+          organizationId,
+          offset,
+          limit: pageLimit,
+          user: {},
+          client: db
+        }
+
+        readUserTeams(job, (err) => {
+          if (err) return cb(err)
+          const pageSize = Math.min(pageLimit, job.user.teams ? job.user.teams.length : 0)
```

Comment:
Is it actually possible for `job.user.teams` to be falsy? Leaving that aside there is no point in counting something that the user can do if he wants.